tos/compat.py

```diff
@@ -50,7 +51,26 @@ def get_cache(cache_name):
         return get_cache(cache_name)
 
 
+def get_middleware_mixin():
+    if django.VERSION >= (1, 10, 0):
+        return deprecation.MiddlewareMixin
+    else:
+        return object
+
+
+def get_middleware_settings_key():
+    if django.VERSION >= (1, 10, 0):
+        return 'MIDDLEWARE'
+    else:
```

Comment:
Based on the [support version](https://www.djangoproject.com/download/#supported-versions), as of April 2020 only **2.2 LTS and newer** are supported.